src/features/singleRight/delegate/ReceiptPage/ActionBarSection/ActionBarSection.tsx

```diff
@@ -31,18 +31,24 @@ export const ActionBarSection = () => {
   });
 
   // logic for finding the delegation with the most failed Delegations
-  delegations.forEach((pd: ProcessedDelegation, index) => {
-    const failedDelegations = pd.bffResponseList?.filter(
-      (data: Right) => data.status !== BFFDelegatedStatus.Delegated,
-    );
+  useEffect(() => {
```

Comment:
No related to your PR but it seems like the text key isn't localized anymore. Seems like something has gone wrong with some merge. Would be sweet if you could update the text key.

<img width="1053" alt="image" src="https://github.com/Altinn/altinn-access-management-frontend/assets/31471142/8a1d5ef2-a89a-4c26-9079-68a49644d467">